app/api/auth/routes.py

```diff
@@ -1,19 +1,116 @@
 from typing import Any
 
-from fastapi import APIRouter, Body, Depends
+from fastapi import APIRouter, Body, Depends, HTTPException, status
 from sqlalchemy.orm import Session
 
 from app import schemas
 from app.api import deps
+from app.crud.base import CRUDBase
+from app.models import Doctor, Branch
+from app.schemas.doctor import CreateDoctor, DoctorUpdate, CreateBranch, BranchUpdate, BaseDoctor, BaseBranch
+from app.api.deps import get_db
 
 router = APIRouter()
 
+doctors_router = APIRouter()
+branches_router = APIRouter()
+
+crud_doctor = CRUDBase(Doctor)
+crud_branch = CRUDBase(Branch)
+
 
 @router.post("/login/access-token", response_model=schemas.Token)
 def login_access_token(
-    db: Session = Depends(deps.get_db), user_data: schemas.UserLogin = Body(...)
+        db: Session = Depends(deps.get_db), user_data: schemas.UserLogin = Body(...)
 ) -> Any:
     """
     OAuth2 compatible token login, get an access token for future requests
     """
     ...
+
+
+@doctors_router.post("/doctors/", response_model=CreateDoctor, status_code=status.HTTP_201_CREATED)
```

Comment:
This is wrong place for this routers, you need to create separate directory called doctors, and put related to doctors endpoints there, in auth module you need to have just auth routers